extensions/wikia/AdEngine/js/lookup/services.js

```diff
@@ -98,8 +99,18 @@ define('ext.wikia.adEngine.lookup.services', [
 		return slotPrices;
 	}
 
+	function storeRealSlotPrices(slotName) {
+		realSlotPrices[slotName] = getCurrentSlotPrices(slotName);
+	}
+
+	function getRealSlotPrices(slotName) {
```

Comment:
Thanks for this suggestion. I tried to find a better name for this method but I failed. `getDfpSlotPrices` sounds great for me.